exercises/beer-song/inputdata.txt

```diff
@@ -0,0 +1,7 @@
+99 bottles of beer on the wall, 99 bottles of beer.\nTake one down and pass it around, 98 bottles of beer on the wall.\n
+44 bottles of beer on the wall, 44 bottles of beer.\nTake one down and pass it around, 43 bottles of beer on the wall.\n
```

Comment:
Yeah, I downloaded it and was playing with it, and as it works, the single lines (one per each test) seems like the way to do it.